openqa-investigate

```diff
@@ -59,14 +59,20 @@ clone() {
 
     # clone job, possibly restarting dependencies
     out=$("${client_call[@]}" -X POST "jobs/$id/restart" 'clone=0' "prio=$((base_prio + prio_add))" "${restart_settings[@]}")
-    handled_ids=$(echo "$out" | runjq '.result[] | keys[] | tonumber')
+    handled_ids=($(echo "$out" | runjq '.result[] | keys[] | tonumber'))
```

Comment:
```In openqa-investigate line 62:
    handled_ids=($(echo "$out" | runjq '.result[] | keys[] | tonumber'))
                 ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```